plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/labeling/helper/StyledTextCalculationHelper.xtend

```diff
@@ -216,7 +224,12 @@ class StyledTextCalculationHelper {
 	 */
 	def dispatch StyledString dispatchGetStyledText(TField tfield) {
 		var styledText = getLabelProvider.getSuperStyledText(tfield);
-		if (tfield.typeRef !== null && styledText !== null) {
+		if (styledText === null) return unknown;
```

Comment:
Similar to fields, getters and setters can also be optional. We need to add `?` to getters and setters if needed as well